src/content/configuration/module.mdx

```diff
@@ -107,6 +107,51 @@ module.exports = {
 };
 ```
 
+
+### module.parser.javascript
+
+Configure options for JavaScript parser.
+
+```js
+module.exports = {
+  module: {
+    parser: {
+      javascript: {
+        // ...
+        commonjsMagicComments: true,
+      },
+    },
+  },
+};
+```
+
+It's allowed to configure those options in [`Rule.parser`](/configuration/module/#ruleparser) as well to target specific modules.
+
+#### module.parser.javascript.commonjsMagicComments
+
+Enable [magic comments](/api/module-methods/#magic-comments) support for CommonJS.
+
+- Type: `boolean`
```

Comment:
We usually put the types below the title, Why this change?

Reply:
It's explicit, and I believe it's much readable. Moreover, with explicit format we could possibly customize those complex types with [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) element in the future.